website/source/docs/internals/providers/lifecycle.html.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: "docs"
+page_title: "Terraform Lifecycle"
+sidebar_current: "docs-internals-provider-guide-lifecycle"
+description: |-
+  Information about the lifecycle your provider fits into.
+---
+
+# Terraform Lifecycle
+
+Terraform runs follow a predictable lifecycle:
```

Comment:
What is a run?

(I know, but it'd be helpful to provide a bit more context here, since we usually say "plan" or "apply" exlicitly)